command/info.go

```diff
@@ -78,15 +98,35 @@ func newInfoEvent(t domain.Issue) flyte.Event {
 	}
 }
 
-type infoSuccessPayload struct {
-	Id          string `json:"id"`
-	Summary     string `json:"summary"`
-	Status      string `json:"status"`
-	Description string `json:"description"`
-	Assignee    string `json:"assignee"`
+func parseInput(input json.RawMessage) (string, error) {
+	log.Printf("Input raw: %s", input)
+	var in string
+	if err := json.Unmarshal(input, &in); err != nil {
+		return "", err
+	}
+
+	// Slack places URLs between <> tags, if using it as the input method
+	// then the tags need to be stripped first before processing the rest
+	if strings.Contains(in, "<") {
+		in = in[1:len(in)-1]
```

Comment:
This is weird code and could have unintended consequences. Your code is saying if the string looks like `foo<bar` then we should change it to `oo<ba`. Better to use startswith, or maybe a regular expression that looks for `<([^>]+)>` and pulls out the capture group?

Reply:
Agreed. It was a hacky way of parsing the string without complicating the existing code too much. In the end I took it out and decided to just use regex to get the ID from a string.